libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/RoadObjectsStore.kt

```diff
@@ -11,14 +17,17 @@ import com.mapbox.navigation.core.trip.model.eh.mapToEHorizonObjectProvider
 import com.mapbox.navigation.core.trip.model.eh.mapToEHorizonObjectType
 import com.mapbox.navigation.core.trip.model.eh.mapToOpenLRStandard
 import com.mapbox.navigation.navigator.internal.MapboxNativeNavigator
+import com.mapbox.navigator.OpenLRLocation
 
 /**
  * [MapboxNavigation.roadObjectsStore] provides methods to get road objects metadata, add and remove
  * custom road objects.
  */
 class RoadObjectsStore internal constructor(
     private val navigator: MapboxNativeNavigator,
+    private val logger: Logger
 ) {
+    private val TAG = Tag("ROAD_OBJECTS_STORE")
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/3976#issuecomment-785934709

```suggestion
    private val TAG = Tag("MbxRoadObjectsStore")
```